vcd/resource_vcd_rde_interface_test.go

```diff
@@ -0,0 +1,135 @@
+//go:build rde || ALL || functional
```

Comment:
I think you're missing `rde` tag  in:
* config_test.go
* provider_test.go


```
go test -tags rde -v -timeout 200m
# github.com/vmware/terraform-provider-vcd/v3/vcd [github.com/vmware/terraform-provider-vcd/v3/vcd.test]
./config_test.go:840:3: undefined: testAccProvider
./config_test.go:841:3: undefined: testAccProviders
./config_test.go:843:12: undefined: testAccProvider
./config_test.go:1277:11: undefined: testAccProvider
./config_test.go:1300:11: undefined: testAccProvider
./config_test.go:1518:15: undefined: createTemporaryVCDConnection
./datasource_vcd_rde_interface_test.go:26:2: undefined: testParamsNotEmpty
./datasource_vcd_rde_interface_test.go:39:22: undefined: buildMultipleProviders
./resource_vcd_rde_interface_test.go:23:2: undefined: testParamsNotEmpty
./resource_vcd_rde_interface_test.go:46:22: undefined: testAccProviders
./resource_vcd_rde_interface_test.go:46:22: too many errors
FAIL    github.com/vmware/terraform-provider-vcd/v3/vcd [build failed]
```

Reply:
Thanks, I forgot to check in this repo, should be fixed now.